src/Compilers/Core/Portable/Operations/ILiteralExpression.cs

```diff
@@ -13,10 +13,6 @@ namespace Microsoft.CodeAnalysis.Semantics
     /// </remarks>
     public interface ILiteralExpression : IOperation
     {
-        /// <summary>
-        /// Textual representation of the literal.
-        /// </summary>
-        string Text { get; }
```

Comment:
I mean, it's not empty. It has all the standard members of IOperation. Text was basically get providing a shortcut on `literalExpression.Syntax.ToString()`, which we don't feel worth having.